lms/views/api/exceptions.py

```diff
@@ -140,6 +141,7 @@ def external_request_error(self):
         return self.error_response(
             message=message,
             details={
+                "cause": str(self.context.__cause__),
```

Comment:
@seanh Should this be `repr` instead?

Reply:
I think it should probably be `repr`, yes. The result depends on the individual exception class (and what it has for its `__str__()` and `__repr__()` methods) but in general `__repr__()` is the one that's supposed to contain more info so that's probably what we want for debugging. If you compare `str()` and `repr()` for common standard library exception classes `str()` will usually return just `'foo'` whereas `repr()` will give `ValueError('foo')`. I think we probably want to know the exception class, otherwise we may not be able to tell what went wrong